conf.d/main

```diff
@@ -39,6 +49,11 @@ for information on using the wiki software.<br><br>
 * [http://lists.wikimedia.org/mailman/listinfo/mediawiki-announce MediaWiki release mailing list]
 " | php $WEBROOT/maintenance/edit.php -u $ADMIN_NAME --no-rc -b Main_Page
 
+# update logo to tkl one
+mv /usr/local/src/logo.png $WEBROOT/resources/assets/
+chown www-data:www-data $WEBROOT/resources/assets/logo.png
+sed -i "\|^$wgLogos|s|wiki.png|logo.png|" $WEBROOT/LocalSettings.php
```

Comment:
This isn't bad per-se but I personally don't like this kind of chained sed command without spacing or grouping, I find it a bit hard to read. My personal preference would be

(with a space)
`"\|^$wgLogos| s|wiki.png|logo.png|"`

Or even with curly brackets
`"\|^$wgLogos|{s|wiki.png|logo.png|}"`

Reply:
I'll implement the space I think... :smile: